handler/handler_test.go

```diff
@@ -78,14 +112,85 @@ func clientMock(doFunc func(ctx context.Context, request *http.Request) (*http.R
 	}
 }
 
-func TestDataImporterHandle(t *testing.T) {
+// Scenario 1
+// a) Create bulk request succeeded with no failed resources
+// b) Update bulk request not made
+func TestHandleWithTwoEventsBothEventCreated(t *testing.T) {
+
+	var count int
+	Convey("Given a handler configured with sucessful es updates for all two events is success", t, func() {
+
+		doFuncWithValidResponse := func(ctx context.Context, req *http.Request) (*http.Response, error) {
+			count++
```

Comment:
As this only makes 1 request, we only need to handle 1 response - so for this one we can reduce the amount of logic in the inline function

Reply:
fyi - still include the count to track if it suddenly gets called twice